src/gui/optionsdialog.cpp

```diff
@@ -1100,18 +1100,18 @@ void OptionsDialog::loadOptions()
         m_ui->checkMaxRatio->setChecked(false);
         m_ui->spinMaxRatio->setEnabled(false);
     }
-    if (session->globalMaxSeedingMinutes() >= 0) {
+    if (session->globalMaxSeedingMinutes().count() >= 0) {
         // Enable
         m_ui->checkMaxSeedingMinutes->setChecked(true);
         m_ui->spinMaxSeedingMinutes->setEnabled(true);
-        m_ui->spinMaxSeedingMinutes->setValue(session->globalMaxSeedingMinutes());
+        m_ui->spinMaxSeedingMinutes->setValue(session->globalMaxSeedingMinutes().count());
```

Comment:
I think a slightly safer way of doing this is to explicitly cast the duration to `minutes` before getting `count()`. Otherwise you rely on the function to specifically return minutes. you can use `duration_cast<minutes>()`